src/github.com/getlantern/flashlight/util/common.go

```diff
@@ -0,0 +1,62 @@
+package util
+
+import (
+	"errors"
+	"fmt"
+	"github.com/getlantern/golog"
+	"io/ioutil"
+	"os"
+	"os/user"
+)
+
+var (
+	log = golog.LoggerFor("util")
+)
+
+type logger func(arg interface{})
+
+// Based on https://github.com/getlantern/lantern-go/blob/master/src/lantern/config/config.go
+
+// determineConfigDir() determines where to load the config by checking the
+// command line and defaulting to ~/.lantern.
+func DetermineConfigDir() (string, error) {
+	usr, err := user.Current()
+	if err != nil {
+		fmt.Errorf("Error location user home directory %v", err)
+		return "", err
+	}
+	return usr.HomeDir + "/.lantern", nil
+}
+
+func Check(e error, log logger, msg string) {
+	if e != nil {
+		log(msg)
+	}
+}
+
+func FileExists(fileName string) (bool, error) {
```

Comment:
Let's not worry about it.  I introduced [appdir] in another branch (used for logging) and I'll refactor this to use it once merged.